### PR TITLE
[MU4] Fix #8253: Wrong shortcuts in MacOS Tooltips

### LINKED ISSUE #8253: [MU4 Issue] Wrong shortcuts in MacOS Tooltips
**Describe the bug**
MacOS tooltips are showing 'Ctrl' instead of 'Cmd' - for example, Cmd + Z is for undo in Mac, and Ctrl + Z for Windows.

**To Reproduce**
1. Open Score
2. Hover over Undo button
3. Locate tooltip text

**Expected behavior**
Replace Ctrl to Cmd

**Screenshots**
image

**Desktop (please complete the following information):**
 - Mac OS

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/8253

### CODE REVIEW

File: src/framework/ui/uitypes.h [hunk 1]
```diff
@@ -282,11 +283,7 @@ struct MenuItem : public UiAction
 
     QString shortcutsAsString() const
```
Comment (igorkorsukov) [inline src/framework/ui/uitypes.h:284]: May be better rename to something like `shortcutsTitle`

File: src/project/view/templatepaintview.h [hunk 2]
```diff
@@ -49,6 +49,8 @@ private slots:
 private:
     void onNotationSetup() override;
 
+    QString sequenceByActionCode(const actions::ActionCode& code) const;
```
Comment (igorkorsukov) [inline src/project/view/templatepaintview.h:52]: May be better rename to something like `shortcutsTitleByActionCode`

**Review by igorkorsukov — APPROVED** [review]

File: src/framework/ui/uitypes.h (hunk 1 above)
Comment (RomanPudashkin) [reply src/framework/ui/uitypes.h]: renamed

File: src/project/view/templatepaintview.h (hunk 2 above)
Comment (RomanPudashkin) [reply src/project/view/templatepaintview.h]: renamed

**Comment by DmitryArefiev** [comment]
Tested #8253 on Mac12 - FIXED      

Please merge into master

**Pull request merged**